docs/UserGuide.md

```diff
@@ -331,13 +351,13 @@ Clears all recipes from the Recipe Book.
 1. `clear recipe`
     <br />Expected Output: <br />
 ![clearRecipe1.png](images/recipeCommands/clearRecipe1.png)
-
-### 4.3.4 Edit Recipes
+   
+### 4.5.4 Edit Recipes
 
 Edit a recipe from the Recipe Book.
 
-**Format:**<br />
-`edit recipe <index> <field flag><new data>...`
+**Format**:<br /> 
+`edit recipe <index> <field flag> <new data>...`
```

Comment:
The proper notation should be 

`edit recipe <index> [-n <name>] [-i <ingredient>...] [-s <step>...] [-d <description>]`